aiida/orm/nodes/data/array/cube.py

```diff
@@ -0,0 +1,246 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""
+Module of the GaussianCubeData class, defining the AiiDA data type for storing
+Gaussian cube files.
+"""
+import os
+import sys
+import numpy as np
+
+from .array import ArrayData
+
+
+class GaussianCubeData(ArrayData):
+    """Class that allows managing gaussian cube files."""
+
+    def __init__(  # pylint: disable=too-many-arguments
+        self,
+        comment=None,
+        voxel=None,
+        origin=None,
+        atomic_numbers=None,
+        atomic_charges=None,
+        atomic_coordinates=None,
+        data=None,
+        data_units=None,
+        **kwargs
+    ):
+        super().__init__(**kwargs)
+        if comment:
```

Comment:
These checks should all be `is not None`. I'm not sure if any of the inputs can be valid when they are `False`-y, but better to be safe than sorry.